Add hash tag based navigation for large  list of menu items.

sameerast opened the pull request:
When the user manual has a long length of the menu items and after scrolling all the way down, if the user clicked on any help menu item which is bottom the list. The page will be refreshed to the new help section but clicked menu item all the down(out of the viewpoint), there is no clue about where is the clicked item.

This code bit always keeps the clicked menu item on the viewpoint.

Commit:
Add hash tag based based navigation for large  list of menu items.

RobErskine commented:
Hey @sameerast. Thanks so much for this PR. I was a little busy today and didn't get a chance to review this. I'll check it out tomorrow for sure though. Thanks again.

RobErskine commented on src/assetbundles/usermanual/dist/css/UserManual.css:
```diff
@@ -175,3 +175,21 @@
+}
+
+#content{
+  position: fixed !important;
```
When I look at this with `position: fixed;` I can't scroll the main content when the content is longer than the main window. Is there a reason this needs fixed instead of relative?

RobErskine commented on src/assetbundles/usermanual/dist/css/UserManual.css:
```diff
@@ -175,3 +175,21 @@
 }
+
+
+//  style to mange hash tag links
```
Sorry to be a stickler, but can we remove the comments?

sameerast replied:
Changes applied.

RobErskine commented on src/assetbundles/usermanual/dist/css/UserManual.css:
```diff
@@ -175,3 +175,21 @@
+}
+
+
+.peUserManual #user-manual-menu{
```
Would it be okay swapping out camelCase for kebab-case to be consistent with the rest of the styles?
`.peUserManual` -> `.pe-user-manual`?

sameerast replied:
Changes applied.

RobErskine reviewed (commented):
Sorry about this, but just a few minor comments before we merge in. Happy to do this on our end if you're not up for it.

Thanks again for your contribution. We really appreciate all the hard work that goes into this.

sameerast committed:
Update UserManual.css

comments removed and kebab case used.

sameerast committed:
Update UserManual.css

If it exceeds the viewpoint height, automatically vertical scroll will be applied.

sameerast committed:
Update UserManual.js

class name changed.

sameerast commented:
Hi Rob, 
I addressed all the points were highlighted. Thank you for taking some time.

sameerast commented:
@RobErskine did you get a chance to review the changes.

RobErskine committed:
Merge branch 'master' into hash-tag-link

RobErskine commented:
Might be over a year later but it's never to late for a merge 😅. Thanks for this @sameerast
